runtime/vm/UpcallVMHelpers.cpp

```diff
@@ -34,6 +34,10 @@ extern "C" {
 
 #if JAVA_SPEC_VERSION >= 16
 
+/* The mask for the normalized type identifier */
+#define J9_FFI_UPCALL_BYTE_TYPE_MASK 0xFF
+#define J9_FFI_UPCALL_SHORT_TYPE_MASK 0xFFFF
```

Comment:
The the comment and the names of these constants are misleading: this has nothing to do with type identifiers and there's no hint that they're only relevant for unsigned values. I suggest the constant literals should just be used instead (in the one location where each is used).

Reply:
Fixed as suggested above.